tests/scripts/thread-cert/node.py

```diff
@@ -135,7 +135,7 @@ def _launch_docker(self):
         assert launch_ok
 
         cmd = f'docker exec -i {self._docker_name} ot-ctl'
-        self.pexpect = pexpect.popen_spawn.PopenSpawn(cmd, timeout=10)
+        self.pexpect = pexpect.popen_spawn.PopenSpawn(cmd, timeout=15)
```

Comment:
```suggestion
        self.pexpect = pexpect.popen_spawn.PopenSpawn(cmd, timeout=30)
```
why not even larger?